faster_whisper/transcribe.py

```diff
@@ -361,9 +362,12 @@ def generate_segments(
         prompt_reset_since = 0
 
         if options.initial_prompt is not None:
-            initial_prompt = " " + options.initial_prompt.strip()
-            initial_prompt_tokens = tokenizer.encode(initial_prompt)
-            all_tokens.extend(initial_prompt_tokens)
+            if isinstance(options.initial_prompt, Iterable):
```

Comment:
I think `str` is also an iterable. To be safe, can you swap the if-else and check for `str` first?

Reply:
Oops. you are correct. Making the change!